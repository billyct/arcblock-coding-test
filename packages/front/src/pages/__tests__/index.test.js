import {render} from '@testing-library/react'

jest.mock('../../components/Search', () => 'search-component')

afterAll(() => {
  jest.resetModules()
})

test('IndexPage Component', () => {
  const IndexPage = require('../index').default
  const {container} = render(<IndexPage/>)
  expect(container).toMatchSnapshot()
})
