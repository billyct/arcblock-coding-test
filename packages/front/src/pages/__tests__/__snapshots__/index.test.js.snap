// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage Component 1`] = `
<div>
  <div
    class="container mx-auto flex flex-col justify-center min-h-screen"
  >
    <div
      class="text-center mb-12"
    >
      <svg
        class="inline-block"
        height="104"
        width="90"
      >
        logo.svg
      </svg>
      <div
        class="text-2xl mt-4 leading-loose font-serif"
      >
        BlockChain Explorer
      </div>
    </div>
    <search-component />
  </div>
</div>
`;
